test/cbmc/proofs/DHCPv6/DHCPv6HandleOption/DHCPv6HandleOption_harness.c

```diff
@@ -0,0 +1,76 @@
+/* Standard includes. */
+#include <stdint.h>
+
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "task.h"
+#include "semphr.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_BitConfig.h"
+#include "FreeRTOS_Sockets.h"
+#include "FreeRTOS_IP_Private.h"
+#include "FreeRTOS_UDP_IP.h"
+#include "FreeRTOS_DHCP.h"
+
+/* CBMC includes. */
+#include "cbmc.h"
+
+
+BaseType_t __CPROVER_file_local_FreeRTOS_DHCPv6_c_prvDHCPv6_handleOption( struct xNetworkEndPoint * pxEndPoint,
+                                                                          uint16_t usOption,
+                                                                          const DHCPOptionSet_t * pxSet,
+                                                                          DHCPMessage_IPv6_t * pxDHCPMessage,
+                                                                          BitConfig_t * pxMessage )
+{
+    return nondet_BaseType();
+}
+
```

Comment:
Please add comment here to describe the reason to wrap `prvDHCPv6_handleStatusCode` out.